boards/shields/esp_8266/doc/index.rst

```diff
@@ -76,16 +81,57 @@ to works:
         - ESP-8266 bootloader won't send garbage. Try connect at 74880 bps if
           you module have 26MHz crystal to detect boot fails.
 
-Programming
-***********
+.. note::
+	The shield configuration was designed to be used with multiple network
+	interfaces.  This means, boards with any other network interface can
+	be simultaneous enabled.  User must be check network documentation to
+	avoid problem when running samples.  For instance, boards with ethernet
+	automatically define that interface as default.  Make sure to select
+	WiFi interface as the default interface when build examples.
+	As general rule, keep only WiFi interface enabled at
+	Networking -> Link Layer Options.  This will ensure samples works as
+	expected.
```

Comment:
I don't quite understand this note:
You first mention that shield is designed to work with multiple network interfaces and you end up stating that they should be disabled.

Reply:
reworked.